lib/addons/p5.dom.js

```diff
@@ -308,6 +308,87 @@
   };
 
   /**
+   * Creates a checkbox &lt;input&gt;&lt;/input&gt; element in the DOM.
+   *
+   * @method createCheckbox
+   * @param  {String} [label] label displayed after checkbox
+   * @param  {boolean} [value] value of the checkbox; checked is true, unchecked is false
+   * @return {Object/p5.Element} pointer to p5.Element holding created
+   *                           node
+   */
+  p5.prototype.createCheckbox = function() {
+    var elt = document.createElement('input');
+    elt.type = 'checkbox';
+    //checkbox must be wrapped in p5.Element before label so that label appears after
+    var self = addElement(elt, this);
+    self.checked = function(){
+      if (arguments.length === 0){
+        return elt.checked;
```

Comment:
do these need to be `self.elt`? or does the reference to `elt` stay bound somehow? I'm assuming you tested and it works fine. it might be good to make the explicit `self.elt` call even if this does work, just to be clear what's happening.